news/2020-06-03-new-featured-artist-noisia.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Noisia"
+date: 2020-06-03 08:00:00 +0000
+---
+
+Make some noise for our latest Featured Artist of EDM fame, **Noisia**!
+
+![](https://assets.ppy.sh/artists/84/header.jpg)
+
+When the Netherlands-based trio of Nik Roos, Thijs de Vlieger and Martijn van Sonderen aren't making waves in the electronic music scene as the maverick act known as **Noisia**, they're creating tunes for big-name games like *Devil May Cry* and *DJ Hero*.
+
+Turning heads and taking names since 2000, we're super excited to welcome **Noisia** aboard with a small selection of some of their recent original works. If you're not familiar with them from their work, you've undoubtedly heard some absolutely filthy bass inspired by their musical mastery.
+
+Don't take my word for it, check them out:
+
+### Vigilantes
+
+Check out [this collaborative set](https://osu.ppy.sh/beatmapsets/765338) hosted by [Affirmation](https://osu.ppy.sh/users/6186628)!
```

Comment:
the beatmap link for Vigilantes should be https://osu.ppy.sh/beatmapsets/1126238/ instead tho, right now it's lnking to Nowa's Thank You Scientist beatmap from last week's FA announcement instead

Reply:
good catch! must've missed it when copying across